src/Request/RequestHandler.php

```diff
@@ -192,6 +193,11 @@ public function request(string $method, string $path, array $parameters = [])
         $options = [
             'headers' => [
                 'Authorization' => $this->getAuthToken(),
+                'User-Agent' => sprintf(
+                  'Clarifai PHP (https://github.com/darrynten/clarifai-php);v%s;%s',
+                  \DarrynTen\Clarifai\Clarifai::VERSION,
```

Comment:
Thanks for the PR!

Is this exactly how you guys want the UA to look?

I'm wondering how this would work if someone wanted to send their own UA through for whatever reason. Would that ever be a use-case?

If not then this would do nicely, if it is a use-case then we can build it to support it now rather than later.

Cheers